package.json

```diff
@@ -18,7 +18,8 @@
     "test:coverage": "nyc npm run test:unit",
     "lint:src": "tslint --format stylish -p tsconfig.json",
     "lint:unit": "tslint -c tslint-test.json --format stylish test/unit/*.ts test/unit/**/*.ts",
-    "lint:integration": "tslint -c tslint-test.json --format stylish test/integration/*.ts"
+    "lint:integration": "tslint -c tslint-test.json --format stylish test/integration/*.ts",
+    "postinstall": "node ./env-check.js"
```

Comment:
We should do a run time check instead of the installation time check. This check will always return true since we are running it with node.

Reply:
Done